anaconda.py

```diff
@@ -291,16 +291,16 @@ def start_debugger(signum, frame):
         flags.dirInstall = True
 
     # Set up logging as early as possible.
-    import logging
-    from pyanaconda import anaconda_log
-    anaconda_log.init()
-    anaconda_log.logger.setupVirtio()
+    from pyanaconda import anaconda_logging
```

Comment:
Why the `anaconda_logging` doesn't provide us loggers?

I have a bad feeling for typos in `anaconda_logging` and `anaconda_loggers`.